gateways/istio-egress/values.yaml

```diff
@@ -56,20 +56,6 @@ gateways:
       secretName: istio-egressgateway-ca-certs
       mountPath: /etc/istio/egressgateway-ca-certs
 
-    sds:
-      # If true, ingress gateway fetches credentials from SDS server to handle TLS connections.
-      enabled: false
-      # SDS server that watches kubernetes secrets and provisions credentials to ingress gateway.
```

Comment:
Maybe this is a copy paste error from the ingress, as you can see all the comments are for ingress